reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,92 @@
+* All components of Anbox Cloud upgrade to NVIDIA 550 driver series by default on Ubuntu 22.04 images. For Ubuntu 20.04 images, 535 driver series will be used. <!--AC-2309-->
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Page level guided tours are now available for help in navigating through the dashboard.
+* Support for using custom authentication server instead of Ubuntu SSO.
+* Custom instance name field in *Create instance* form.
+* User experience enhancements to the AAOS control panel:
+  - Extending VHAL support to allow setting the geographic location and to be able to provide and revoke consent for enabling APIs in the browser when using maps.
+* Enhancements to forms to support AOSP vs AAOS applications.
+
+### Other
+
+* Support for building custom images of Anbox Cloud based on AAOS. See [Custom images](tbd).<!--AC-2458-->
+* Android security updates for May 2024 (see [Android Security Bulletin - May 2024](https://source.android.com/docs/security/bulletin/2024-05-01) for more information).<!--AC-2437-->
+* The Android WebView has been updated to [124.0.6367.82](https://chromereleases.googleblog.com/2024/04/chrome-for-android-update_24.html).
+
+## Deprecations
+
+* Support for Ubuntu 20.04 (focal) will be dropped in a future release.
+* Support for the EmuGL renderer is removed in Anbox Cloud 1.23.0.
+* The Anbox Cloud Appliance is being reworked and the current implementation is deprecated. Existing installations of the current implementation will receive updates until Anbox Cloud 1.24.2. Starting from 1.23.0, new installations will receive the new implementation of the appliance.
+
+  The new implementation of the appliance will no longer use the Juju charmed operators internally but package all necessary service components directly within the snap. This helps to simplify the installation process and reduce overall footprint.
+
+## Removed functionality
+
+* Documentation on how to enable VirGL manually is removed because with this release, VirGL is the default renderer for NVIDIA GPUs.
+
+## Known issues
+
+The following known issues exist in the 1.22.0 release and are planned to be fixed in future releases:
+
+* Video playback does not work on 64-bit only systems where the NVIDIA GPU is installed.
+* Creating instances with streaming enabled using the `enable-streaming` option requires manually adding the new relation between the AMS and the stream agent charms. See [How to upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox).
```

Comment:
Routing people to [How to upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) where generally how to upgrade *regular* deployment by adding the relation between ams and stream-agent doesn't fit well for the appliance. This issue is anbox cloud *appliance* specific, and the relation must be established automatically when upgrading the Anbox Cloud appliance to 1.22.0(while it's not the case for regular deployment where manual setup is always required in such a case).
Hence, I think we need to call out here that for the **appliance**, people need to manually add the relation between ams and stream agent to work around the known issue. @jat-canonical could you please put a small guide here in the release notes please.
Thanks